src/Popover/PopoverButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { StyledButton } from './Popover.styles';
+
+export interface ButtonProps {
+  onClick: () => void;
+  style?: React.CSSProperties; // Can we set this globally somewhere?
```

Comment:
I think you can just extend ReactHTMLButonProps?

Reply:
That's much better, cheers